docs/source/installation.rst

```diff
@@ -38,7 +38,7 @@ MParT uses CMake to handle dependencies and compiler configurations.   A basic b
    cd build
    cmake                                               \
      -DCMAKE_INSTALL_PREFIX=<your/MParT/install/path>  \
-     -DKokkos_ENABLE_THREADS=ON                        \
+     -DKokkos_ENABLE_PTHREAD=ON                        \
```

Comment:
So I originally changed this because Kokkos broke compatibility of `KOKKOS_ENABLE_PTHREAD` with the release of 3.7.0 and changed it to [`KOKKOS_ENABLE_THREADS`](https://kokkos.github.io/kokkos-core-wiki/ProgrammingGuide/Compiling.html?highlight=kokkos_enable_threads#general-information). Should we add a tip for this as well?